hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -169,8 +170,9 @@ public class HoodieWriteConfig extends HoodieConfig {
   public static final ConfigProperty<String> PRECOMBINE_FIELD_NAME = ConfigProperty
       .key("hoodie.datasource.write.precombine.field")
       .noDefaultValue()
-      .withDocumentation("Field used in preCombining before actual write. When two records have the same key value, "
-          + "we will pick the one with the largest value for the precombine field, determined by Object.compareTo(..)");
+      .withDocumentation("Comma separated list of fields used in preCombining before actual write. When two records have the same key value, "
```

Comment:
wondering if we can rename the config to "hoodie.write.ordering.field" 
and then add old config value as alternate key.

Reply:
The field is not just used for write, it is also a read config I think, actually it should be a table config instead of a write config to avoid ambiguties, maybe "hoodie.table.ordefing.fields".

Reply:
@lokeshj1703 the config name should def. become plural now as per this PR. Also agree with @danny0405  to make it a table config (this will be consistent with record key fields and partition fields)